app/src/androidTest/java/org/mozilla/fenix/ui/TabbedBrowsingTest.kt

```diff
@@ -74,7 +72,6 @@ class TabbedBrowsingTest {
         }.openHomeScreen { }
 
         homeScreen {
-            mDevice.wait(Until.findObject(By.res("R.id.item_tab")), TestAssetHelper.waitingTime)
```

Comment:
I think most of the deleted mDevice.waits might be all necessary.  The failure I see in Firebase seems to be because it's trying to search for the test result too fast

Reply:
If I recall, in general, we only added mDevice waits when there was originally a timing issue of some kind, so I think No-Jun may be right.

Reply:
Restored. Changed the selector to a different one because FindObject is unable to find the CardView resource name `R.id.item_tab` for whatever reason.

Reply:
@AaronMT oh, the resource name is not `R.id.item_tab`for sure.  it's what that refers to, you can see it in uiautomatorviewer.  it's probably something like `org.mozilla.fenix.debug:id/item_tab` .  R.id.item_tab is just a reference to that string.  We can't use it like that.